crates/sui-core/src/authority.rs

```diff
@@ -1315,27 +1315,43 @@ impl AuthorityState {
                 .expect("Creating an executor should not fail here");
 
         let expensive_checks = false;
-        let (inner_temp_store, effects, _execution_error) = executor
-            .execute_transaction_to_effects(
-                &self.database,
-                protocol_config,
-                self.metrics.limits_metrics.clone(),
-                expensive_checks,
-                self.certificate_deny_config.certificate_deny_set(),
-                &epoch_store.epoch_start_config().epoch_data().epoch_id(),
-                epoch_store
-                    .epoch_start_config()
-                    .epoch_data()
-                    .epoch_start_timestamp(),
-                input_objects,
-                gas_object_refs,
-                gas_status,
-                kind,
-                signer,
-                transaction_digest,
-            );
+        let (inner_temp_store, effects, execution_error) = executor.execute_transaction_to_effects(
+            &self.database,
+            protocol_config,
+            self.metrics.limits_metrics.clone(),
+            expensive_checks,
+            self.certificate_deny_config.certificate_deny_set(),
+            &epoch_store.epoch_start_config().epoch_data().epoch_id(),
+            epoch_store
+                .epoch_start_config()
+                .epoch_data()
+                .epoch_start_timestamp(),
+            input_objects,
+            gas_object_refs,
+            gas_status,
+            kind,
+            signer,
+            transaction_digest,
+        );
+
         let tx_digest = *effects.transaction_digest();
 
+        info!(
+            "gas_stats {}: err = {:?}, o_s = {}, o_c = {}, \
+                    o_m = {}, o_u = {}, o_d = {}, \
+                    o_u_d = {}, o_w = {}, sgn = {}",
+            tx_digest,
+            execution_error.err(),
+            effects.input_shared_objects().len(),
+            effects.created().len(),
+            effects.mutated().len(),
+            effects.unwrapped().len(),
+            effects.deleted().len(),
+            effects.unwrapped().len(),
+            effects.wrapped().len(),
+            signer,
+        );
```

Comment:
Based on my understanding of what you want, my recommendation would be to explicitly set the [target](https://docs.rs/tracing/latest/tracing/index.html#configuring-attributes) in these "gas stats" log lines and then locally utilize the existing tracing infrastructure to configure log files as well as using the RUST_LOG env var in order to explicitly collect these gas stats lines.

Reply:
Ok, I will add the target that way, thanks! 

In terms of the second part of your recommendation, I wonder if you are suggesting to remove the layer from the telemetry config? If so, it seems that by relying on RUST_LOG to filter out these lines in production, it would be more burdensome overall, and I am not sure how to have them off by default without creating the layer as I did.  I am perhaps misunderstanding.

What do you mean also by using existing tracing infrastructure to configure log files?

Reply:
>In terms of the second part of your recommendation, I wonder if you are suggesting to remove the layer from the telemetry config? If so, it seems that by relying on RUST_LOG to filter out these lines in production, it would be more burdensome overall, and I am not sure how to have them off by default without creating the layer as I did. I am perhaps misunderstanding.

Yes i am suggesting removing the layer from the telemetry config. It wouldn't be burdensome in production to not have these lines show up. (As things are today with these two PRs, these lines will show up in prod and cause issues). In order to not have this be an issue a combination of setting a custom `target` as well as potentially having the log level be at the `trace` level. Then to enable this all you need to do is `RUST_LOG=gas_stats=trace` and otherwise these lines wont show up in prod

Reply:
Sounds like I can close the other PR then if I can make the above recommendation work locally

Reply:
RUST_LOG *is* a mechanism for specifying a filter, so the correct filter would just need to be provided.

Reply:
It seems you are strongly advocating for not adding a layer to the telemetry config and to use RUST_LOG to do all the configuration. I now see that it is likely fully possible to achieve the desired outcome using the RUST_LOG variable. However, I would like for when anyone comes to ask me "How to I generate gas stats logs?" to be able to respond, all you need to do is set the output file in the NodeConfig, rather than to say RUST_LOG=sui=trace, [but not info] filter="..what was that string again?", file="something" etc, which requires people, (including me), to need to remember and/or dig through the code and sparse documentation on the proper RUST_LOG syntax, check what level the gas stats are at, etc. Having the layer will also make it much easier to trigger this log output programmatically from something like a stress test as well.

Reply:
@bmwill suggestion makes sense to me and @laura-makdah and I talked about it and we want to try to move forward with that model! 
We will ahve more soon